src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/ControllerAdvice.kt

```diff
@@ -161,7 +161,7 @@ class ControllerAdvice {
 
   @ExceptionHandler(Exception::class)
   fun handleException(e: Exception): ResponseEntity<ErrorResponse> {
-    log.error("Unexpected exception {}", e.message)
+    log.error("Unexpected exception", e)
```

Comment:
want the full stack trace if unexpected exception otherwise can't find the cause